src/modules/externalUrls.ts

```diff
@@ -16,15 +16,6 @@ export const XDAI_GAS_STATION = `https://blockscout.com/xdai/mainnet/api/v1/gas-
 export const getBlockscoutUserURL = (userAddress: string) =>
   `https://blockscout.com/xdai/mainnet/address/${userAddress}/transactions`;
 
-/*
- * Coin Machine
- */
-export const CM_LEARN_MORE = `https://colony.gitbook.io/colony/extensions/coin-machine`;
-export const CM_GET_WHITELISTED = `https://colony.gitbook.io/colony/get-started/how-to-get-whitelisted-for-the-clny-sale.`;
-export const CM_BLOG_POST = `https://blog.colony.io/introducing-coin-machine/`;
-export const CM_GOOGLE_SHEET = `https://docs.google.com/spreadsheets/d/1ZCuFcwqI4S6ZK5OwTl1yN7AK8mjv5d_V3g-_kMen01Y/edit#gid=2013814210`;
-export const CM_DESCRIPTION = 'https://colony.gitbook.io/colony/';
```

Comment:
Maybe ask @arrenv or however is managing our Gitbook, if these links are still relevant to them, otherwise they should remove them on their end as well.

Reply:
Good point, there is some content we will still want to use, so, I have temporarily added a note on each of the Coin Machine pages in GitBook.